lib/avo/base_resource.rb

```diff
@@ -16,6 +16,7 @@ class BaseResource
     class_attribute :includes, default: []
     class_attribute :model_class
     class_attribute :translation_key
+    class_attribute :translation_enable
```

Comment:
This being a boolean, wouldn't a better name be `translation_enabled` (ending in `d`)?

Reply:
Good call! made the change